evap/contributor/templates/contributor_course_form.html

```diff
@@ -65,9 +69,9 @@ <h4 class="modal-title">Preview</h4>
     {% trans "Approve course" as title %}
     {% trans "Do you want to approve this course? This will allow the student representatives to proceed with the preparation, but you won't be able to make any further changes." as question %}
     {% trans "Approve course" as action_text %}
-    {% include "confirmation_modal.html" with modal_id="approve_course_modal" title=title question=question action_text=action_text btn_type="primary" %}
+    {% include "confirmation_modal.html" with modal_id="approveCourseModal" title=title question=question action_text=action_text btn_type="primary" %}
     <script type="text/javascript">
-        function approve_course_modal_action(data_id) {
+        function approveCourseModal_action(data_id) {
```

Comment:
javascript usually has camel case. the mixed style was used because the `_action`and `show_` parts are how our modal code creates method names. in the old version i had some difficulties understanding which object is what because of these automatically generated names and this made it a lot easier in my opinion. i see your point but am not completely happy with the snake case we had.